packages/ui5-proxy-middleware/src/base/utils.ts

```diff
@@ -242,7 +238,7 @@ export async function resolveUI5Version(version?: string, log?: ToolsLogger): Pr
  */
 export const injectUI5Url = async (htmlFilePath: string, ui5Configs: ProxyConfig[]): Promise<string | undefined> => {
     if (existsSync(htmlFilePath)) {
-        let html = await promises.readFile(htmlFilePath, { encoding: 'utf8' });
+        let html = readFileSync(htmlFilePath, { encoding: 'utf8' });
```

Comment:
If we are going to use `readFileSync`, then the function `injectUI5Url` doesn't need to be defined as `async`.

Reply:
https://github.com/SAP/open-ux-tools/pull/424/commits/229211b2df005d465452fb9dd06715c21a8d32a0